packages/story-editor/src/app/pageDataUrls/pageDataUrlsProvider.tsx

```diff
@@ -41,24 +43,30 @@ function PageDataUrlProvider({ children }) {
    * @param {Page} storyPage Page object.
    * @return {Function} function to cancel image generation request
    */
-  const queuePageImageGeneration = useCallback(
-    (storyPage) => {
-      const idleTaskUid = storyPage.id;
-      const idleTask = async () => {
-        const dataUrl = await storyPageToDataUrl(storyPage, {});
-        setDataUrls((state) => ({
-          ...state,
-          [storyPage.id]: dataUrl,
-        }));
-      };
+  const queuePageImageGeneration: PageDataUrlsActions['queuePageImageGeneration'] =
+    useCallback(
+      (storyPage: Page) => {
```

Comment:
Does it still require adding `Page` here even when using `PageDataUrlsActions['queuePageImageGeneration']` above?

Reply:
Removed --- your right it wasn't needed.